src/provider/application.js

```diff
@@ -251,7 +252,7 @@ class Application extends EventEmitter {
 
     // Emit a ready event
     this.emit('xfc.ready');
-    this.JSONRPC.notification('authorized', [{ url: window.location.href }]);
+    this.JSONRPC.notification('authorized', [{ url: window.location.href, options: this.options }]);
```

Comment:
Should we always [send options ](https://github.com/cerner/xfc/pull/46/files#diff-74eec411683c67343a77d00160da3589R212)or should we only send if it has something?

Reply:
Is it non-passive?

Reply:
it is just extra info object always present/received, it could be read or ignored.

Reply:
I have no problem with always sending it  then

Reply:
Is any work required to make this available on the Consumer?

Reply:
We don't need any changes on Consumer as we are merging in the new options object with url into one detail object. https://github.com/cerner/xfc/blob/master/src/consumer/frame.js#L51